kmod/src/ioctl.h

```diff
@@ -454,11 +457,14 @@ struct scoutfs_ioctl_alloc_detail_entry {
  * EISDIR: either the source or destination is a directory.
  * ENODATA: either the source or destination file have offline extents.
  */
+#define SCOUTFS_IOC_MB_STAGE		(1 << 0)
```

Comment:
When we add flags we need to add error handling to return errors if unknown flag bits are set.  We've done it a few ways in a few places, I think I'd try _UKNOWN (U64_MAX << 1) after the _STAGE definition.  Then test flags & _UNKNOWN in the ioctl and return EINVAL.